lib/utils.js

```diff
@@ -40,12 +40,16 @@ async function killAllSimulators (timeout = DEFAULT_SIM_SHUTDOWN_TIMEOUT) {
   const appName = xcodeVersion.major >= 7 ? 'Simulator' : 'iOS Simulator';
 
   // later versions are slower to close
-  timeout = timeout * (xcodeVersion.major === 8 ? 2 : 1);
+  timeout = timeout * (xcodeVersion.major >= 8 ? 2 : 1);
 
-  let pids;
+  try {
+    await exec('xcrun', ['simctl', 'shutdown', xcodeVersion.major > 8 ? 'all' : 'booted'], {timeout});
+  } catch (ign) {}
```

Comment:
we don't actually verify if any simulator is running there before executing shutdown, so it will always show a log message in such case, which might be confusing

Reply:
Ok. Maybe that's why it was never logged before.